src/MLOps.NET.SQLServer/MLOpsBuilderExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿using MLOps.NET.SQLServer.Storage;
+using MLOps.NET.Storage;
+
+namespace MLOps.NET.SQLServer
+{
+    /// <summary>
+    /// Extension methods to allow the usage of SQL Server
+    /// </summary>
+    public static class MLOpsBuilderExtensions
+    {
+        /// <summary>
+        /// Enables the usage of SQL Server
+        /// </summary>
+        /// <param name="builder">MLOpsBuilder for using SQL Server</param>
+        /// <param name="connectionString"></param>
+        /// <returns>Provided MLOpsBuilder for chaining</returns>
+        public static MLOpsBuilder UseSQLServer(this MLOpsBuilder builder, string connectionString)
+        {
+            builder.UseMetaDataStore(new SQLServerMetaDataStore(new DbContextFactory(connectionString)));
```

Comment:
I don't like the fact that we need to explicitly create the factory, can we just pass in the `connectionString` and create factory in Metadata constructor?

Reply:
Absolutely, I think that's a valid point. Let me push a change for that.

Reply:
Actually, the reason why I'm doing it like that is so I can mock away the IDbContextFactory in unit tests for the SQLMetaDataStore, so I would rather keep it as is.